pkgs/development/python-modules/python-louvain/default.nix

```diff
@@ -13,10 +16,19 @@ buildPythonPackage rec {
     sha256 = "sha256-t7ot9QAv0o0+54mklTK6rRH+ZI5PIRfPB5jnUgodpWs=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/taynaud/python-louvain/pull/95/commits/c95d767e72f580cb15319fe08d72d87c9976640b.patch";
```

Comment:
Suggest giving this patch a `name = `

Reply:
How would you name that patch? `karate.patch`?

Reply:
```suggestion
      # fix karake tests
      url = "https://github.com/taynaud/python-louvain/pull/95/commits/c95d767e72f580cb15319fe08d72d87c9976640b.patch";
```

Reply:
done!

Reply:
What is does - `fix-karate-test-networkx-2.7.patch`

Reply:
Added that :+1: